onecodex/viz/_metadata.py

```diff
@@ -171,35 +192,43 @@ def plot_metadata(
         if ylabel is None:
             ylabel = magic_fields[vaxis]
 
-        alt_kwargs = {}
+        encode_kwargs = {}
         if facet_by:
-            alt_kwargs["column"] = alt.Column(
+            encode_kwargs["column"] = alt.Column(
                 facet_by, header=alt.Header(titleOrient="bottom", labelOrient="bottom")
             )
 
+        x_kwargs = {"axis": alt.Axis(title=xlabel)}
+        if secondary_haxis:
+            x_kwargs.update(
+                {
+                    "title": None,
+                    "axis": alt.Axis(title=xlabel, labels=False, ticks=False),
```

Comment:
Why are we setting `labels` and `ticks` explicitly to `False` here and not above?

Reply:
This is part of the hack to make a faceted plot look like a grouped plot: https://stackoverflow.com/a/66877669/3776794

If we don't turn off x-axis labels and ticks it looks like this (the x-axis labels are redundant with the coloring and tooltip):

<img width="784" alt="Screenshot 2024-02-15 at 8 11 51 AM" src="https://github.com/onecodex/onecodex/assets/1847232/65b21f49-8856-42bb-9c86-ecfaabec81bc">

Reply:
I'll add a comment to the code explaining why we're doing this